templates/skeleton/app/routes/account.status.tsx

```diff
@@ -0,0 +1,12 @@
+import type {ShouldRevalidateFunction} from '@remix-run/react';
+import {type LoaderFunctionArgs, json} from '@remix-run/server-runtime';
+
+export const shouldRevalidate: ShouldRevalidateFunction = ({formAction}) => {
+  return !!formAction?.startsWith('/account');
+};
+
+export async function loader({context}: LoaderFunctionArgs) {
+  return json({
+    isLoggedIn: await context.customerAccount.isLoggedIn(),
+  });
+}
```

Comment:
Shouldn't this be part of the example then since we are not using it in the template? Or we want to make it easier to switch to this from our template?

Reply:
This is added to both the skeleton and the example. Since it's a diff example, should we only add it in one place?